packages/@aws-cdk/aws-scheduler-targets-alpha/lib/sqs-send-message.ts

```diff
@@ -0,0 +1,69 @@
+import { ISchedule, IScheduleTarget, ScheduleTargetConfig } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import * as sqs from 'aws-cdk-lib/aws-sqs';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * Properties for a SQS Queue Target
+ */
+export interface SqsSendMessageProps extends ScheduleTargetBaseProps {
+  /**
+   * The FIFO message group ID to use as the target.
+   *
+   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-scheduler-schedule-sqsparameters.html#cfn-scheduler-schedule-sqsparameters-messagegroupid
+   */
+  readonly messageGroupId?: string;
+}
+
+/**
+ * Use an Amazon SQS Queue as a target for AWS EventBridge Scheduler.
+ */
+export class SqsSendMessage extends ScheduleTargetBase implements IScheduleTarget {
+  constructor(
+    private readonly queue: sqs.IQueue,
+    private readonly props: SqsSendMessageProps,
+  ) {
+    super(props, queue.queueArn);
+
+    if (props.messageGroupId !== undefined) {
+      if (props.messageGroupId.length < 1 || props.messageGroupId.length > 128 ) {
+        throw new Error(`messageGroupId length must be between 1 and 128, got ${props.messageGroupId.length}`);
+      }
+      if (!queue.fifo) {
+        throw new Error('target must be a FIFO queue if messageGroupId is specified');
+      }
```

Comment:
CFn occurs a following error when the queue is not FIFO queue if messageGroupId is specified. So added this validation.

```
ROLLBACK_COMPLETE: Resource handler returned message: "Invalid request provided: Parameters MessageGroupId not valid for the target.
```